docs/appendix/glossary/gantry.md

```diff
@@ -2,8 +2,8 @@
 title: Gantry
 id: attribute
 full_link: /components/gantry/
-short_description: A robot that only uses linear motion to carry out a task; for example, the scaffolding of a 3D printer, which moves the print head around on motorized linear rails.
+short_description: A mechanical system that only uses linear motion to carry out a task.
 aka:
 ---
 
-A robot that only uses linear motion to carry out a task; for example, the scaffolding of a 3D printer, which moves the print head around on motorized linear rails.
+A mechanical system that only uses linear motion to carry out a task; for example, the scaffolding of a 3D printer, which moves the print head around on motorized linear rails.
```

Comment:
ahhhh *motorized* linear rails...a different solution to the old gantry clarification stuff :)